builder-support/dockerfiles/Dockerfile.rpmsbom

```diff
@@ -1,8 +1,10 @@
-# No SBOM on el-7, which only has Python 2 and will be EOL soon anyway
+ARG BUILDER_VERSION
+ARG BUILDER_TARGET
 
+# No SBOM on el-7, which only has Python 2 and will be EOL soon anyway
 RUN touch /var/lib/rpm/* && \
     if ! $(grep -q 'release 7' /etc/redhat-release); then \
-      yum upgrade -y && yum install -y python3 python3-dnf; \
```

Comment:
Was it intended to skip the upgrade?

Reply:
Yes, because otherwise `yum` might try to downgrade the installed packages!